### PR TITLE
add format title on hover label

### PR DESCRIPTION
**Pull request by zinchenkoivan**
## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

no opportunity format title on hover label

## What is the new behavior?
add format title on hover label

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

Screenshot 2024-04-23 at 13 05 13

### CODE REVIEW

File: src/common/Axis/RadialAxis/RadialAxisTickSeries/RadialAxisTickLabel.tsx [hunk 1]
```diff
@@ -59,6 +59,11 @@ export interface RadialAxisTickLabelProps
    * Format of the label.
    */
   format?: (value: any, index: number) => any | string;
+
+  /**
+   * Format title on hover label.
+   */
+  formatHoverTitle?: (value: any, index: number) => any | string;
```
[the remark this answers is not in the document]
Comment (zinchenkoivan) [reply src/common/Axis/RadialAxis/RadialAxisTickSeries/RadialAxisTickLabel.tsx:66]: this is an additional feature for format, on hover state format label

[the remark this answers is not in the document]
Comment (amcdnl) [reply src/common/Axis/RadialAxis/RadialAxisTickSeries/RadialAxisTickLabel.tsx:66]: How about `formatTooltip` ? If we do this, we should add it to LinearAxisLabel too.

**Comment by zinchenkoivan** [comment]
Screenshot 2024-05-03 at 16 28 36

File: src/common/Axis/LinearAxis/LinearAxisTickLabel.tsx [hunk 2]
```diff
@@ -149,14 +154,15 @@ export const LinearAxisTickLabel: FC<Partial<LinearAxisTickLabelProps>> = ({
 
   const { x, y } = getOffset();
   const textPosition = getTextPosition();
+  const titleHover = formatTooltip ? formatTooltip(fullText) : fullText;
```
Comment (amcdnl) [inline src/common/Axis/LinearAxis/LinearAxisTickLabel.tsx:157]: You probably need to check if its a function ... typeof formatTooltip === function

**Review by amcdnl — APPROVED** [review]

**Pull request merged**